sphinx/application.py

```diff
@@ -871,6 +873,9 @@ def add_js_file(self, filename: str, **kwargs: str) -> None:
 
             app.add_js_file('example.js', async="async")
             # => <script src="_static/example.js" async="async"></script>
+            
```

Comment:
Oh, flake8 warns for this line. Could you remove spaces from here please?

Reply:
done!